sources/osg/Material.js

```diff
@@ -96,6 +96,15 @@ define( [
             return this._shininess;
         },
 
+        setShadeless: function ( a ) {
```

Comment:
Just added get/set over existing _shadeless
Was indeed surprised to find the _shadeless member in Material here....